apps/alpha-test-app/src/views/LayoutDemoView.vue

```diff
@@ -1,453 +1,134 @@
-        Tooltip parent
-        <div class="tooltip__text">Hello world Hello world Hello world Hello world</div>
-      </div>
-    </div>
-
-    <!----------- GRID main (right col) ----------->
-    <div class="page" :class="{ 'page--full-height': !settings.footer.showFullFooter }">
-      <div class="page__content">
-        <OnyxHeadline is="h1">Scrollable page content</OnyxHeadline>
-
-        <LayoutSettings v-model="settings" horizontal />
-        <!-- <LayoutDemoOptions /> -->
-        <!-- demo sticky content -->
-        <div v-if="settings.content.showStickyContent" class="demo sticky-content">
-          Sticky content. Example: Breadcrumb bar, Table header, Headlines
-        </div>
+const isSidebarOpen = ref(true);
 
-        <!-- demo flyout -->
-        <p>
-          <label>
-            Drop Down:
-            <span class="flyout-parent">
-              <input @click="settings.content.showFlyout = !settings.content.showFlyout" />
-              <div v-if="settings.content.showFlyout" class="demo flyout">
-                <LayoutSettings v-model="settings" :show="['content']" />
-              </div>
-            </span>
-          </label>
-        </p>
-
-        <!-- demo tooltip -->
-        <div class="demo tooltip" :class="{ 'tooltip--forced': settings.content.forceTooltip }">
-          Tooltip parent
-          <div class="tooltip__text">Hello world Hello world Hello world Hello world</div>
-        </div>
+const showTempSidebarOpen = computed<boolean>(() => {
+  const { showTempOverlayTransparent, showTempOverlay } = settings.value.sidebar;
+  return (isSidebarOpen.value && (showTempOverlay || showTempOverlayTransparent)) || false;
+});
 
-        <!-- demo page content -->
-        <template v-if="settings.content.showLongPageContent">
-          <p v-for="content in muchContent" :key="content">{{ content }}</p>
-        </template>
-      </div>
-    </div>
+const footerAsideSidebar = computed<boolean>(
+  () => (settings.value.footer.showDetailFooter && isSidebarOpen.value) || false,
+);
+</script>
 
-    <!----------- GRID bottom row ----------->
-    <div
-      v-if="settings.footer.showDetailFooter || settings.footer.showFullFooter"
-      class="demo footer bottom-bar"
-      :class="{ 'footer--detail': settings.footer.showDetailFooter }"
+<template>
+  <OnyxAppLayout :nav-bar-alignment="navBarLeft ? 'left' : 'top'">
+    <template #navBar>
+      <NavBarDemo v-model="navBarLeft">
+        <FlyoutDemo v-model="settings.content.showFlyout" :small="navBarLeft">
+          <LayoutSettings v-model="settings" :show="['content']" />
+        </FlyoutDemo>
+        |
+        <TooltipDemo :force-tooltip="settings.content.forceTooltip" style="display: inline-block" />
+      </NavBarDemo>
+    </template>
+
+    <!-- key is not needed in a real application where the slots aren't constantly hidden and revealed -->
+    <OnyxPageLayout
+      :key="JSON.stringify(settings)"
```

Comment:
We should definately find a way so this is not needed...Vue slots should be reactive.
Can you create a follow up ticket for this?